inbm-lib/inbm_common_lib/utility.py

```diff
@@ -205,7 +205,9 @@ def is_within_directory(directory: str, target: str) -> bool:
     return prefix == abs_directory
 
 
-def safe_extract(tarball: tarfile.TarFile, path: str = ".", members: Optional[Iterable[tarfile.TarInfo]] = None, *, numeric_owner: bool = False) -> None:
+def safe_extract(tarball: tarfile.TarFile,
```

Comment:
Ensure that the `safe_extract` function checks for path traversal vulnerabilities when extracting tarball contents. [important]